ledger/ledger-api-domain/src/main/scala/com/digitalasset/ledger/api/domain.scala

```diff
@@ -338,4 +358,9 @@ object domain {
         reason: String,
     ) extends PackageEntry
   }
+
+  object Logging {
+    implicit def `tagged value to LoggingValue`[T: ToLoggingValue, Tag]: ToLoggingValue[T @@ Tag] =
+      value => value.unwrap
+  }
```

Comment:
I do not completely understand the code here yet but I have the feeling this could be quite useful for the json api too :thinking:

Reply:
It says that if we have a type `T`, a type `Tag`, and an instance of `ToLoggingValue[T]`, then we also have an instance of `ToLoggingValue[T @@ Tag]` (which just discards the tag).

For example, if we have a value of type `Ref.TransactionId @@ TransactionIdTag`, we can convert it to a logging value, because we know how to convert `Ref.TransactionId` to a logging value.

Happy to discuss further offline if you'd like to understand more.